connect/runtime/src/main/java/org/apache/kafka/connect/runtime/AbstractHerder.java

```diff
@@ -272,8 +273,10 @@ public ConfigInfos validateConfigs(String connType, Map<String, String> connecto
             }
             ReflectionsUtil.registerUrlTypes();
             ConfigurationBuilder builder = new ConfigurationBuilder().setUrls(ClasspathHelper.forJavaClassPath());
-            Reflections reflections = new Reflections(builder);
 
+            builder.getUrls().remove(ReflectionsUtil.stringtoURL("/"));
```

Comment:
Hmm, removing it here seems like it definitely solves the common case where this is a mistake. I can't really imagine anyone intending to pull in all of `/` unless they have some really tiny distribution, but it feels odd to special case it in the Java code when it's really an issue with how we construct classpaths in our scripts.

More generally, would this even work as expected for Windows?